diff --git a/node_modules/solid-start/api/index.ts b/node_modules/solid-start/api/index.ts
index 91f28a9..a8bd667 100644
--- a/node_modules/solid-start/api/index.ts
+++ b/node_modules/solid-start/api/index.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { registerApiRoutes } from "./internalFetch";
 import { getRouteMatches } from "./router";
 import { MatchRoute, Method, Route } from "./types";
diff --git a/node_modules/solid-start/api/middleware.ts b/node_modules/solid-start/api/middleware.ts
index f36d81c..4e33608 100644
--- a/node_modules/solid-start/api/middleware.ts
+++ b/node_modules/solid-start/api/middleware.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { Middleware } from "../entry-server";
 import { FetchEvent, FETCH_EVENT } from "../server/types";
 import { getApiHandler } from "./index";
diff --git a/node_modules/solid-start/api/router.ts b/node_modules/solid-start/api/router.ts
index cc4eb7a..cbfdf6a 100644
--- a/node_modules/solid-start/api/router.ts
+++ b/node_modules/solid-start/api/router.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { MatchRoute, Method } from "./types";
 
 export function getRouteMatches(routes: MatchRoute[], path: string, method: Method) {
diff --git a/node_modules/solid-start/api/types.ts b/node_modules/solid-start/api/types.ts
index f0b94e2..a5ddba2 100644
--- a/node_modules/solid-start/api/types.ts
+++ b/node_modules/solid-start/api/types.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { FetchEvent, FETCH_EVENT } from "../server/types";
 
 export interface APIEvent extends FetchEvent {
diff --git a/node_modules/solid-start/data/Form.tsx b/node_modules/solid-start/data/Form.tsx
index e908349..2bfe4a9 100644
--- a/node_modules/solid-start/data/Form.tsx
+++ b/node_modules/solid-start/data/Form.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 /*!
  * Original code by Remix Sofware Inc
  * MIT Licensed, Copyright(c) 2021 Remix software Inc, see LICENSE.remix.md for details
diff --git a/node_modules/solid-start/data/FormError.tsx b/node_modules/solid-start/data/FormError.tsx
index f0c79c3..2f34ed1 100644
--- a/node_modules/solid-start/data/FormError.tsx
+++ b/node_modules/solid-start/data/FormError.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 export class ServerError extends Error {
   status: number;
   constructor(message: string, { status, stack }: { status?: number; stack?: string } = {}) {
diff --git a/node_modules/solid-start/data/createRouteAction.tsx b/node_modules/solid-start/data/createRouteAction.tsx
index 91c1ca2..33254b0 100644
--- a/node_modules/solid-start/data/createRouteAction.tsx
+++ b/node_modules/solid-start/data/createRouteAction.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useNavigate, useSearchParams, type Navigator } from "@solidjs/router";
 import { $TRACK, batch, createSignal, useContext } from "solid-js";
 import { FormError, FormImpl, FormProps } from "./Form";
diff --git a/node_modules/solid-start/data/createRouteData.tsx b/node_modules/solid-start/data/createRouteData.tsx
index 348d454..4c7ac4a 100644
--- a/node_modules/solid-start/data/createRouteData.tsx
+++ b/node_modules/solid-start/data/createRouteData.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type {
   Resource,
   ResourceFetcher, ResourceFetcherInfo, ResourceOptions, Signal
diff --git a/node_modules/solid-start/entry-client/StartClient.tsx b/node_modules/solid-start/entry-client/StartClient.tsx
index c17a8e5..53915fa 100644
--- a/node_modules/solid-start/entry-client/StartClient.tsx
+++ b/node_modules/solid-start/entry-client/StartClient.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { MetaProvider } from "@solidjs/meta";
 import { Router, RouterProps } from "@solidjs/router";
 // @ts-ignore
diff --git a/node_modules/solid-start/entry-client/mount.tsx b/node_modules/solid-start/entry-client/mount.tsx
index d4194c1..7584228 100644
--- a/node_modules/solid-start/entry-client/mount.tsx
+++ b/node_modules/solid-start/entry-client/mount.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type { JSX } from "solid-js";
 import { getOwner } from "solid-js";
 import { createComponent, getNextElement, hydrate, render } from "solid-js/web";
diff --git a/node_modules/solid-start/entry-server/StartServer.tsx b/node_modules/solid-start/entry-server/StartServer.tsx
index 2ded03d..97b5f2d 100644
--- a/node_modules/solid-start/entry-server/StartServer.tsx
+++ b/node_modules/solid-start/entry-server/StartServer.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { MetaProvider } from "@solidjs/meta";
 import { RouteDataFunc, Router, RouterProps } from "@solidjs/router";
 import { ComponentProps, sharedConfig } from "solid-js";
diff --git a/node_modules/solid-start/entry-server/render.ts b/node_modules/solid-start/entry-server/render.ts
index 0b70260..1665155 100644
--- a/node_modules/solid-start/entry-server/render.ts
+++ b/node_modules/solid-start/entry-server/render.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { JSX } from "solid-js";
 import { renderToStream, renderToString, renderToStringAsync } from "solid-js/web";
 import { internalFetch } from "../api/internalFetch";
diff --git a/node_modules/solid-start/index.tsx b/node_modules/solid-start/index.tsx
index 74f088b..f73a665 100644
--- a/node_modules/solid-start/index.tsx
+++ b/node_modules/solid-start/index.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 export { Link, Meta, Style, Stylesheet, Title } from "@solidjs/meta";
 export {
   Navigate,
diff --git a/node_modules/solid-start/islands/index.tsx b/node_modules/solid-start/islands/index.tsx
index 5b8ff20..60ce9f9 100644
--- a/node_modules/solid-start/islands/index.tsx
+++ b/node_modules/solid-start/islands/index.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { Component, ComponentProps, lazy, splitProps, useContext } from "solid-js";
 import { Hydration, NoHydration } from "solid-js/web";
 import { ServerContext } from "../server/ServerContext";
diff --git a/node_modules/solid-start/islands/router.ts b/node_modules/solid-start/islands/router.ts
index 6ee641e..9a47639 100644
--- a/node_modules/solid-start/islands/router.ts
+++ b/node_modules/solid-start/islands/router.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type { Location, Navigator } from "@solidjs/router";
 import { createSignal } from "solid-js";
 interface LocationEntry {
diff --git a/node_modules/solid-start/islands/server-router.tsx b/node_modules/solid-start/islands/server-router.tsx
index a778780..9b6e8f8 100644
--- a/node_modules/solid-start/islands/server-router.tsx
+++ b/node_modules/solid-start/islands/server-router.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { createContext, JSX, useContext } from "solid-js";
 import { ssr } from "solid-js/web";
 export interface RouteDefinition {
diff --git a/node_modules/solid-start/root/Document.tsx b/node_modules/solid-start/root/Document.tsx
index 19a9f0a..f3e3a0f 100644
--- a/node_modules/solid-start/root/Document.tsx
+++ b/node_modules/solid-start/root/Document.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type { JSX } from "solid-js";
 import { children, ComponentProps } from "solid-js";
 import { insert, NoHydration, spread, ssrElement } from "solid-js/web";
diff --git a/node_modules/solid-start/root/InlineStyles.tsx b/node_modules/solid-start/root/InlineStyles.tsx
index d215c4b..83114ab 100644
--- a/node_modules/solid-start/root/InlineStyles.tsx
+++ b/node_modules/solid-start/root/InlineStyles.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { Style } from "@solidjs/meta";
 import { createResource, Show, Suspense, useContext } from "solid-js";
 import type { PageEvent } from "../server";
diff --git a/node_modules/solid-start/root/Links.tsx b/node_modules/solid-start/root/Links.tsx
index 04708cc..c4c121d 100644
--- a/node_modules/solid-start/root/Links.tsx
+++ b/node_modules/solid-start/root/Links.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { JSXElement, useContext } from "solid-js";
 import { useAssets } from "solid-js/web";
 import { ServerContext } from "../server/ServerContext";
diff --git a/node_modules/solid-start/root/Scripts.tsx b/node_modules/solid-start/root/Scripts.tsx
index ea6dae4..e4d41bd 100644
--- a/node_modules/solid-start/root/Scripts.tsx
+++ b/node_modules/solid-start/root/Scripts.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { useContext } from "solid-js";
 import { HydrationScript, isServer, NoHydration } from "solid-js/web";
 import { ServerContext } from "../server/ServerContext";
diff --git a/node_modules/solid-start/router.tsx b/node_modules/solid-start/router.tsx
index 56ba963..c6cb446 100644
--- a/node_modules/solid-start/router.tsx
+++ b/node_modules/solid-start/router.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import {
   A as BaseA,
   Location,
diff --git a/node_modules/solid-start/server/ServerContext.tsx b/node_modules/solid-start/server/ServerContext.tsx
index 006547d..02a89a8 100644
--- a/node_modules/solid-start/server/ServerContext.tsx
+++ b/node_modules/solid-start/server/ServerContext.tsx
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { PageEvent } from "./types";
 
 import { createContext, useContext } from "solid-js";
diff --git a/node_modules/solid-start/server/middleware.ts b/node_modules/solid-start/server/middleware.ts
index 610d050..5185ebd 100644
--- a/node_modules/solid-start/server/middleware.ts
+++ b/node_modules/solid-start/server/middleware.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { internalFetch } from "../api/internalFetch";
 import { Middleware as ServerMiddleware } from "../entry-server/StartServer";
 import { ContentTypeHeader, XSolidStartContentTypeHeader, XSolidStartOrigin } from "./responses";
diff --git a/node_modules/solid-start/server/responses.ts b/node_modules/solid-start/server/responses.ts
index 194cb45..4b46bf2 100644
--- a/node_modules/solid-start/server/responses.ts
+++ b/node_modules/solid-start/server/responses.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 export const XSolidStartStatusCodeHeader = "x-solidstart-status-code";
 export const XSolidStartLocationHeader = "x-solidstart-location";
 export const LocationHeader = "Location";
diff --git a/node_modules/solid-start/server/server-functions/server.ts b/node_modules/solid-start/server/server-functions/server.ts
index d09165c..bb76466 100644
--- a/node_modules/solid-start/server/server-functions/server.ts
+++ b/node_modules/solid-start/server/server-functions/server.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { sharedConfig } from "solid-js";
 import { internalFetch } from "../../api/internalFetch";
 import { FormError } from "../../data";
diff --git a/node_modules/solid-start/server/server-functions/types.ts b/node_modules/solid-start/server/server-functions/types.ts
index 78930a0..dd53266 100644
--- a/node_modules/solid-start/server/server-functions/types.ts
+++ b/node_modules/solid-start/server/server-functions/types.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import { FetchEvent } from "../types";
 
 export type ServerFunction<E extends any[], T extends (...args: [...E]) => void> = ((
diff --git a/node_modules/solid-start/session/cookie.ts b/node_modules/solid-start/session/cookie.ts
index aca7311..c80f546 100644
--- a/node_modules/solid-start/session/cookie.ts
+++ b/node_modules/solid-start/session/cookie.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 /*!
  * cookie
  * Copyright(c) 2012-2014 Roman Shtylman
diff --git a/node_modules/solid-start/session/cookies.ts b/node_modules/solid-start/session/cookies.ts
index bfde0bb..17ef465 100644
--- a/node_modules/solid-start/session/cookies.ts
+++ b/node_modules/solid-start/session/cookies.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 /*!
  * Original code by Remix Sofware Inc
  * MIT Licensed, Copyright(c) 2021 Remix software Inc, see LICENSE.remix.md for details
diff --git a/node_modules/solid-start/session/memoryStorage.ts b/node_modules/solid-start/session/memoryStorage.ts
index fac4ebe..8a14ea6 100644
--- a/node_modules/solid-start/session/memoryStorage.ts
+++ b/node_modules/solid-start/session/memoryStorage.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 /*!
  * Original code by Remix Sofware Inc
  * MIT Licensed, Copyright(c) 2021 Remix software Inc, see LICENSE.remix.md for details
diff --git a/node_modules/solid-start/types.ts b/node_modules/solid-start/types.ts
index c44a14c..fecdb58 100644
--- a/node_modules/solid-start/types.ts
+++ b/node_modules/solid-start/types.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type { Debugger } from "debug";
 import type { Component } from "solid-js";
 
